include/aspect/material_model/rheology/strain_dependent.h

```diff
@@ -178,6 +204,15 @@ namespace aspect
            */
           std::vector<double> viscous_strain_weakening_factors;
 
+          /**
+           * The factor of healing time used in the strain healing calculation.
```

Comment:
The wording here seems off. How about the "The healing time scale used in the temperature dependent" strain healing model.